### ISSUE TITLE
Regarding dereplication of bins

### ISSUE THREAD

**Issue by ShailNair** [AnantharamanLab/vRhyme#10]
Hi,
Thanks for the excellent tool. I did binning using vRhyme and PHAMB (VAMB) tools. Next, I want to combine the bins from these two packages into a new non-redundant set. Can I use the **dereplication** option of vRhyme to do this?
If not, do you have any suggestions regarding this aspect?

Thank you

**Comment by KrisKieft** [comment]
The built in dereplication mode for vRhyme isn't necessarily built for combining bins. dRep can dereplicate bins into a non-redundant set, but it doesn't combine to my knowledge. DASTool will combine microbial bins but I think it's dependent on microbial single copy genes. There may not be a best option at the moment. Are you more interest in combining or just dereplicating? The easiest way would be to pick one bin representative instead of combining.

**Comment by ShailNair** [comment]
Yes. combining is a problem with viral bins. I used [the dRep](https://github.com/MrOlm/drep) tool to pick a representative bin based on ANI similarity, contig length, and N50. Although this isn't the best option, I didn't find any other alternative.

**Comment by KrisKieft** [comment]
I don't really have any better alternatives. Maybe the viral group at JGI has a tool in the works.

**Comment by ShailNair** [comment]
That sounds great. hope it comes out soon. Thanks.
